src/app/map/leaflet-map/major-mines-popup/major-mines-popup.component.html

```diff
@@ -0,0 +1,55 @@
+<div class="popup-container">
+  <header class="popup-header">
+    <h5>{{mine.MINE_NAME}}</h5>
+    <a class="popup-close-button" role="button" tabindex="0" (click)="zoomToProject()"><i class="material-icons">zoom_in</i></a>
+    <a class="popup-zoom-button" role="button" tabindex="0" (click)="closePopups()"><i class="material-icons">close</i></a>
```

Comment:
This pulls the close button out of the tab flow. Is there another way to dismiss with a keyboard?

Reply:
Thats a good point. I've changed these to buttons instead of anchors. Tab-able and clickable with the keyboard.